src/main/java/org/grizz/model/User.java

```diff
@@ -14,8 +17,13 @@
 
     @Id
     private String id;
+    @Indexed(unique = true)
+    @Size(min = 8, max = 30)
```

Comment:
It's good to define custom message for validation errors. Default ones are in english only